src/main/java/org/apache/commons/lang3/time/DateUtils.java

```diff
@@ -1175,6 +1196,7 @@ private static Calendar modify(final Calendar val, final int field, final Modify
                             val.add(aField[0], 1);
                         }
                     }
+                    //-1 = 22
                     return val;
```

Comment:
I think that you normally count all returns as a single exit point.

If you imagine it as a graph they all properly leave through the expected exit node. While an exception exits through another node.

Reply:
Thanks! I changed my comment, and then the result matches Lizard.